python/lsst/atmospec/spectraction.py

```diff
@@ -357,33 +372,40 @@ def run(self, exp, xpos, ypos, target, outputRoot=None, plotting=True):
         #     image, xpos, ypos = self.flipImageLeftRight(image, xpos, ypos)
         #     self.displayImage(image, centroid=(xpos, ypos))
 
-        # Use fast mode
         if parameters.CCD_REBIN > 1:
             self.log.info(f'Rebinning image with rebin of {parameters.CCD_REBIN}')
-            # TODO: Fix bug here where the passed parameter isn't used!
+            apply_rebinning_to_parameters()
             image.rebin()
             if parameters.DEBUG:
-                image.plot_image(scale='symlog', target_pixcoords=image.target_guess)
+                self.log.info('Parameters post-rebinning:')
+                dumpParameters()
+                self.debugPrintTargetCentroidValue(image)
+                title = 'Rebinned image with input target location'
+                image.plot_image(scale='symlog', target_pixcoords=image.target_guess, title=title)
```

Comment:
Are these plots safe when running as a pipetask? I know this is a free-floating class, but it's directly called by `ProcessStarTask`, and the new config defaults make me concerned.

Reply:
I think that they're safe, but I agree that the default, to display, is likely wrong. I'll change `doDisplayPlots` to default to `False` - does that sound OK to you?

Reply:
That works for me.